README.md

```diff
@@ -27,15 +42,15 @@ Table of contents
 In the tutorials, we assume the following directory structure:
 
 ```bash
-.
+test_deezy/
 ├── dataset
 │   ├── characters_v001.vocab
 │   └── dataset-string-similarity_test.txt
 └── inputs
     └── input_dfm.yaml
 ```
 
-For this, we first create a test directory (<ins>**note that we strongly recommend creating this directory outside of the DeezyMarch directory cloned and installed following the installation section. After installation, DeezyMatch command lines and modules are accessible from anywhere on your local machine.**</ins>):
+For this, we first create a test directory (<ins>**note that this directory can be created outside of the DeezyMarch source code. After installation, DeezyMatch command lines and modules are accessible from anywhere on your local machine.**</ins>):
```

Comment:
maybe instead of all bold we could use the ⚠️ sign

Reply:
Done